optuna/multi_objective/_hypervolume/base.py

```diff
@@ -17,4 +16,20 @@ def compute(self, solution_set: np.ndarray, reference_point: np.ndarray) -> floa
                 The reference point to compute the hypervolume.
         """
 
+        self._validate(solution_set, reference_point)
+        return self._compute(solution_set, reference_point)
+
+    @staticmethod
+    def _validate(solution_set: np.ndarray, reference_point: np.ndarray) -> None:
+        # Validates that all solution dominates or equal to the reference point.
```

Comment:
```suggestion
        # Validates that all solutions dominate or are equal to the reference point.
```

Reply:
This sentence seems to be [addressed already](https://github.com/optuna/optuna/pull/1537/files#diff-e400f5a3ed3f514c0e87fbbd588f1774R24). The following `ValueError` message could be fixed though, similar to what you suggest, i.e. "All solutions must dominate or be equal to ...".

Reply:
Yes, it is fixed as 
```python
# Validates that all points in the solution set dominate or equal the reference point.
```
Thank you for the comments.